app/assets/javascripts/discourse/app/components/plugin-outlet.js

```diff
@@ -78,12 +79,22 @@ export default class PluginOutletComponent extends GlimmerComponentWithDeprecate
     return result;
   }
 
-  get connectors() {
-    return renderedConnectorsFor(
+  @bind
+  getConnectors({ hasBlock }) {
+    const connectors = renderedConnectorsFor(
```

Comment:
Sorry I know it's draft, so maybe you are in the process of changing it, but  what's the point of managing an array if we accept only one? Why not just have `this.connector`  and check of set?

Reply:
`plugin-outlet` still needs to support having multiple connectors in the majority of places. We're only limiting it to one for 'wrapper plugin outlets' (which have a default implementation in core). We identify the wrapper ones by checking for `has-block`

Reply:
ok I see 👌